app/assets/stylesheets/left_nav.scss

```diff
@@ -0,0 +1,13 @@
+$rock: #685C53;
+$beige: #D4C99E;
+
+#left_nav_index {
```

Comment:
Avoid using id selectors<br>Selector `left_nav_index` should be written in lowercase with hyphens